util/packer/ubuntu-14.04/scripts/install.sh

```diff
@@ -138,7 +138,7 @@ enable_cgroups() {
 create_groups() {
   groupadd docker
   groupadd fuse || true
-  usermod -a -G docker,fuse "${SUDO_USER}"
+  usermod -a -G docker,fuse,libvirtd "${SUDO_USER}"
```

Comment:
This runs before installing libvirt, so we should probably create the group first like the other ones.